packages/ods-react/src/components/tree-view/src/components/tree-view-node/TreeViewNode.tsx

```diff
@@ -0,0 +1,208 @@
+import { TreeView as VendorTreeView } from '@ark-ui/react/tree-view';
+import classNames from 'classnames';
+import { type ComponentPropsWithRef, type FC, type JSX, type KeyboardEvent, type ReactNode, forwardRef, useRef } from 'react';
+import { Checkbox, CheckboxControl } from '../../../../checkbox/src';
+import { ICON_NAME, Icon } from '../../../../icon/src';
+import { useTreeView } from '../../contexts/useTreeView';
+import { useTreeViewNodes } from '../../contexts/useTreeViewNodes';
+import style from './treeViewNode.module.scss';
+
+export interface TreeViewItem<CustomData = Record<string, never>> {
```

Comment:
declare in context to avoid cross-component type import and risk of circular import